sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -660,3 +704,140 @@ def count_overload(df, axis=0, level=None, numeric_only=False):
     ser_par = {'level': 'level'}
 
     return sdc_pandas_dataframe_reduce_columns(df, name, params, ser_par)
+
+
+def sdc_pandas_dataframe_isin_iter_codegen(df, values, func_name, ser_param):
+    all_params = ['df', 'values']
+
+    df_func_name = f'_df_{func_name}_impl'
+    func_text, global_vars = _dataframe_apply_columns_codegen(func_name, all_params, ser_param, df.columns)
+
+    loc_vars = {}
+    exec(func_text, global_vars, loc_vars)
+    _reduce_impl = loc_vars[df_func_name]
+    return _reduce_impl
+
+
+def sdc_pandas_dataframe_isin_ser_codegen(df, values, func_name):
```

Comment:
Could you please separate code generation and receiving python object part?

So we would have function which returns generated function code as text and global vars needed to it.